### PR TITLE
add welcome device-message, show saved-messages

### PR DESCRIPTION
**Pull request by r10s**
counterpart on android: https://github.com/deltachat/deltachat-android/pull/1121

### CODE REVIEW

File: deltachat-ios/DC/Wrapper.swift [hunk 1]
```diff
@@ -157,6 +158,28 @@ class DcContext {
...
+
+    @discardableResult
+    func addDeviceMessage(label: String, msg: DcMsg) -> Int {
+        return Int(dc_add_device_msg(contextPointer, label.cString(using: .utf8), msg.cptr))
+    }
+
+    func addWelcomeDeviceMessages() {
+        // create the device-chat by adding a message to it
+        let msg1 = DcMsg(type: DC_MSG_TEXT)
+        msg1.text = "Welcome to Delta Chat! – Delta Chat looks and feels like other popular messenger apps, "
```
Comment (cyBerta) [inline deltachat-ios/DC/Wrapper.swift]: Why are you hardcoding the string here?

**Comment by cyBerta** [comment]
besides the open question here: https://github.com/deltachat/deltachat-ios/pull/413/files#r352537770, it looks good to me. 
Personally, I would probably encapsulate the core calls in a DcMsg object, but that's not a big deal.

File: deltachat-ios/DC/Wrapper.swift (hunk 1 above)
Comment (r10s) [reply deltachat-ios/DC/Wrapper.swift]: it's just now. once the text is settled, it will go to transifex as usual.

**Comment by r10s** [comment]
turned this into wip, we decided yesterday that the welcome messasges etc. are added by the core, the ui can add messages on their own, however, just now this seems not to be needed.

so, we'll use sth. as dc_update_device_messages() that will do most of the things, see https://github.com/deltachat/deltachat-core-rust/issues/937

**Pull request merged**